tools/config/__init__.py

```diff
@@ -1058,9 +1062,29 @@ def validate_config(self):
 
         Arguments: None
         """
+
         params, _ = self.get_config_data()
+        err_msg = ""
+
+        for param in params:
```

Comment:
```suggestion
        for param in sorted(params):
```
Should fix the ordering issues, if params can be sorted.

But params is a dict, yeah?
```suggestion
        for name, param in sorted(params.items()):
```